src/style.css

```diff
@@ -0,0 +1,122 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.body {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.form {
+  margin: 20px auto 0;
+  display: flex;
+  justify-content: center;
+}
+
+.form__big {
+  width: 100%;
+  height: 70px;
+}
+
+.form__small {
+  width: 100%;
+  height: 42px;
+}
+
+.form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+  /* height: 100%; */
+}
+
+.form__label:after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.form__label_big:after {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 26px;
+  left: 25px;
+}
+
+.form__label_small:after {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.form__input {
+  width: 100%;
+  height: 100%;
+  color: #3d4e61;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+  font-weight: 300;
+  /* font-family: inherit; */
+}
+
+.form__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form__input_big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
```

Comment:
you can move this styles to .form__input, and only create .form__input--small, than you'll have one base class - .form__input and one where you will override needed styles .form__input--small